fact-bounty-client/src/components/common/NavBar/index.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import MenuIcon from "@material-ui/icons/Menu";
+import SwipeableDrawer from "../SwipeableDrawer";
+import PublicLinks from "./PublicLinks";
+import PrivateLinks from "./PrivateLinks";
+import Toast from "../Toast";
+import { logoutUser } from "../../../redux/actions/authActions";
+
+class NavBar extends Component {
```

Comment:
https://github.com/scorelab/fact-Bounty/blob/f570ab4c70ccc2661fb97fd138bec8d31afb0072/fact-bounty-client/src/components/common/NavBar/index.js#L16-L88

Move X component out of the index.js and have it as X.js. 
index.js should be used for exporting modules.